graylog2-server/src/main/java/org/graylog2/bootstrap/preflight/web/resources/PreflightResource.java

```diff
@@ -79,10 +79,19 @@ public List<DataNode> listDataNodes() {
 
         return activeDataNodes.values().stream().map(n -> {
             final var preflight = preflightDataNodes.get(n.getNodeId());
-            return new DataNode(n.getNodeId(), n.getType(), n.getTransportAddress(), preflight != null ? preflight.state() : null, preflight != null ? preflight.errorMsg() : null, n.getHostname(), n.getShortNodeId());
+            return new DataNode(n.getNodeId(), n.getType(), removeCredentialsFromTransportAddress(n.getTransportAddress()), preflight != null ? preflight.state() : null, preflight != null ? preflight.errorMsg() : null, n.getHostname(), n.getShortNodeId());
```

Comment:
Aren't there more places where you have to hide credentials?
(like ClusterResource? )

Reply:
I made a note in our project board so we fix this in a separate PR

Reply:
Will be solved in #16055